tests/installation/installation_overview.pm

```diff
@@ -62,6 +62,27 @@ sub ensure_ssh_unblocked {
     }
 }
 
+sub check_default_target {
+    # Check the systemd target where scenario make it possible
+    return if (is_caasp || is_upgrade || is_hyperv ||
+        get_var('REMOTE_CONTROLLER') || (get_var('BACKEND', '') =~ /spvm|pvm_hmc/));
+    # exclude non-desktop environment as well
+    return unless get_var('DESKTOP');
+
+    # Set expectations
+    my $expected_target = check_var('DESKTOP', 'textmode') ? "multi-user" : "graphical";
+
+    select_console 'install-shell';
+
+    # default.target is not yet linked, so we parse logs and assert expectations
+    if (my $log_line = script_output("grep '$target_search' /var/log/YaST2/y2log | tail -1")) {
```

Comment:
Keep in mind that as per http://open.qa/api/testapi/#_script_output test will fail in case return code is not zero. So we should add `proceed_on_failure` not to fail in case grep didn't find anything.

Reply:
good catch, I didn't use that before, thanks. fixed.